testsuite/cluster-test/src/experiments/performance_benchmark_three_region_simulation.rs

```diff
@@ -39,6 +41,10 @@ impl ExperimentParam for PerformanceBenchmarkThreeRegionSimulationParams {
 
 #[async_trait]
 impl Experiment for PerformanceBenchmarkThreeRegionSimulation {
+    fn affected_validators(&self) -> HashSet<String> {
+        instance::instancelist_to_set(self.cluster.validator_instances())
```

Comment:
I wonder if we actually need to add it? It is still good to have health checks on validators under `ThreeRegionSimulation` experiment, since we don't expect any breakages with extra network latency, and it worked well so far.

Reply:
Shouldn't need it. Removed.